bedrock/firefox/templates/firefox/os/faq.html

```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
-        </dl>
-      </div>
-    </section>
-    <section>
-      <h2 data-accordion-role="tab">{{_('Privacy and Security')}}</h2>
-      <div data-accordion-role="tabpanel">
-          <p>
-          {{_('Mozilla was the first to implement the Do Not Track privacy feature.')}}
-          {% trans url='https://support.mozilla.org/kb/how-turn-do-not-track-feature-firefox-os' %}
-          Firefox also features customizable security settings to manage passwords, history, private data, cookies, site options and add-ons. <a href="{{url}}">Learn more.</a>
-          {% endtrans %}
-          </p>
-      </div>
-    </section>
-    <section>
-      <h2 data-accordion-role="tab">{{_('Developer Questions')}}</h2>
-      <div data-accordion-role="tabpanel">
-          <p>{% trans url='https://developer.mozilla.org/docs/Mozilla/Firefox_OS' %}
-          Are you a developer? Have questions? <a href="{{url}}">Learn more at the Mozilla Developer Network.</a>
-          {% endtrans %}</p>
-      </div>
-    </section>
-  </div>
-
+    <div id="faq" class="accordion accordion-auto-init zebra">
+        <section>
+            <h2 data-accordion-role="tab">{{_('About Firefox OS')}}</h2>
+            <div data-accordion-role="tabpanel">
+                <dl class="faq">
+                    <dt>{{_('What is Firefox OS?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        {{_('Firefox OS is the only truly open mobile OS that unlocks the power of the Web as the platform and enables operators, hardware manufacturers and developers to create innovative and customized applications and products for consumers to use across these connected devices.')}}
+                    {% else %}
+                        {{_('Backed by some of the biggest names in the industry, Firefox OS is a new open operating system for smartphones developed by Mozilla.')}}
+                        {{_('Built entirely using HTML5 and other open Web standards, Firefox OS is free from the rules and restrictions of existing proprietary platforms.')}}
+                    {% endif %}
+                    </dd>
+
+                    <dt>{{_('How is Firefox OS different from other operating systems?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        <p>
+                            <b>{{_('For users')}}</b> {{_('Firefox OS offers a customized and easy to use experience appropriate to consumers from the value range in India to early tech adopters in Japan')}}
+                        </p>
+                        <p>
+                            <b>{{_('For operators and OEMs')}}</b> {{_('Firefox OS offers the opportunity to capture new user segments due to its open development process and tight collaboration with partners searching for new revenue streams and business models ')}}
```

Comment:
Nit: This sentence and the one before is missing a period at the end of the sentence. This one also has an extra leading space at the end.